__tests__/app.test.js

```diff
@@ -76,9 +76,27 @@ describe('GET /api/', () => {
             })
         })
 
+        test('200: responds with article object with the added property comment_count when passed comment_count query', () => {
+            return request(app)
+            .get('/api/articles/3?comment_count')
```

Comment:
Hey Habiba, 
I think theres been a little confusion for this task, we'd like you to query the db directly and add a comment_count each time a request is made to GET `/api/reviews/:review_id` not as a URL query.

Reply:
Definitely got confused between the two queries, will refactor now. Thanks for the feedback.